ddagent.py

```diff
@@ -261,7 +262,11 @@ def flush(self):
                 tornado.httpclient.AsyncHTTPClient.configure("tornado.curl_httpclient.CurlAsyncHTTPClient")
         else:
             log.debug("Using SimpleHTTPClient")
-        http = tornado.httpclient.AsyncHTTPClient()
+
+        if self._use_blocking_http_client:
```

Comment:
Although this is perfectly valid, you might be able to avoid this (which isn't really something we'd likely use) by using the tornado test classes which have some facilities to test the async stuff: http://www.tornadoweb.org/en/stable/testing.html